executor/runcexecutor/executor.go

```diff
@@ -182,7 +182,7 @@ func (w *runcExecutor) Run(ctx context.Context, id string, root executor.Mount,
 		defer clean()
 	}
 
-	mountable, err := root.Src.Mount(ctx, false)
+	mountable, err := root.Src.Mount(ctx, root.Readonly)
```

Comment:
This change was made because I was trying to enforce that `readonly` always get set to `true` when requesting a `Mountable` for an `ImmutableRef`. Previously, if you requested a mount for a finalized `ImmutableRef` and set `readonly` to false, the returned mount would be read-only anyways (because it's created by `View` underneath). Also, getting a mount for an unfinalized `ImmutableRef` and then making changes to it seems dangerous.

That being said, I'll revert this and the corresponding check in `immutableRef.Mount` for now as fixing it here isn't totally necessary anymore (I believe it was in a previous iteration but I've changed some things since then).